src/MessageEditor.js

```diff
@@ -64,15 +78,23 @@ class MessageEditor extends React.Component {
 
     return (
       <div className="MessageEditor">
-          <TextField
-            id="outlined-password-input"
-            label="Password"
-            type="password"
-            size="small"
-            margin="normal"
-            variant="outlined"
-            onBlur={(event) => {this.setState({ pass: event.target.value })}}
+        <Grid container alignItems="center" spacing={1} style={{'margin-top': '1%', 'margin-bottom': '1%'}}>
+          <Grid item>
+            <TextField
+              id="outlined-password-input"
+              label="Password"
+              type="password"
+              size="small"
+              variant="outlined"
+              onBlur={(event) => {this.setState({ pass: event.target.value })}}
+            />
+          </Grid>
+          <ImportButton
+            addToSnackbar={this.showAlert}
+            pass={this.state.pass}
+            update={this.updateMessages}
```

Comment:
Maybe it could be more efficient to have the backend return what was updated and then we modify state ourselves without having to send another request?